Code.gs

```diff
@@ -38,6 +38,8 @@ var defaultAllDayReminder = -1;        // Default reminder for all day events in
                                        // See https://github.com/derekantrican/GAS-ICS-Sync/issues/75 for why this is neccessary.
 var addTasks = false;
 
+var emailWhenModified = false;  
+var emailWhenAdded = false;  
```

Comment:
Why were these added? These are old and `emailSummary` should be used in any place these used to be